src/tapis-app/Streams/MeasurementsListing/_components/Measurements/Measurements.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styles from './Measurements.module.scss';
+import { v4 as uuidv4 } from 'uuid';
+import MeasurementsPlot from '../MeasurementsPlot';
+
+const Measurements: React.FC<{
+  variable: string;
+  graphWidth: number;
+  id: string;
+  measurements: { [datetime: string]: number };
+  select: () => void;
+  selected: boolean;
+}> = ({ variable, graphWidth, id, measurements, select, selected }) => {
+  let plotlyLayout: Partial<Plotly.Layout> = {
+    width: graphWidth,
+    height: 400,
+  };
+
+  return (
+    <li className={styles.li} onClick={select}>
+      {`${variable}`}
+      <div className={styles['measurements-list']}>
+        {Object.entries(measurements).map((entry: [string, number]) => {
+          let date = entry[0].replace('T', ' ');
+          return <div key={uuidv4()}>{`${date}: ${entry[1]}`}</div>;
```

Comment:
If date is unique, can it be used for the key?

Reply:
I assume that the key would need to be globally unique. There may be (will likely be) other measurements objects with the same datetime element. This will be the case if there are multiple variables tracked by the instrument that have values for the same datetime.